From daf96d31abe500d7c36a1af320df446921cd9a5c Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Wed, 11 May 2022 22:19:45 -0500
Subject: [PATCH 218/261] tests: Add /sbin and /usr/sbin to path in partmap
 test

The partmap test requires no elevated privileges. However, it uses parted
which can be used as a normal user, but is usually located in /sbin or
/usr/bin (eg. on Debian systems). Whereas the normal user does not usually
have /sbin or /usr/sbin added to their path, thus parted will not be found
causing the test to abort. Add /sbin and /usr/sbin to the path for the
partmap test so that the test can run successfully as an unprivileged user.

Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 tests/partmap_test.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/partmap_test.in b/tests/partmap_test.in
index a73c473b0..4138e88fe 100644
--- a/tests/partmap_test.in
+++ b/tests/partmap_test.in
@@ -19,6 +19,8 @@ set -e
 parted=parted
 grubshell=@builddir@/grub-shell
 
+PATH="$PATH:/sbin:/usr/sbin"
+
 . "@builddir@/grub-core/modinfo.sh"
 
 create_disk_image () {
-- 
2.36.1

